FBSimulatorControl/Utility/FBSimulatorLaunchCtl.m

```diff
@@ -141,8 +141,39 @@ - (nullable NSString *)startServiceWithName:(NSString *)serviceName error:(NSErr
   return serviceName;
 }
 
+- (BOOL)restartServiceWithName:(NSString *)serviceName timeout:(NSTimeInterval)timeout error:(NSError **)error
```

Comment:
Could you not use ` - (BOOL)amendRelativeToPath:(NSString *)relativePath defaults:(NSDictionary<NSString *, id> *)defaults managingService:(NSString *)serviceName error:(NSError **)error` to do this instead?